sources/framework/VEModel/R/models.R

```diff
@@ -1285,7 +1285,7 @@ ve.stage.init <- function(Name=NULL,Model=NULL,ScenarioDir=NULL,modelParam_ls=NU
 
   # Identify "startFrom" stage (VEModelStage$runnable will complete setup)
   # Can find StartFrom through ModelStages or from the stage configuration file/parameters
-  if ( !is.character(self$StartFrom) || length(self$StartFrom)==0 || ! nzchar(self$StartFrom) ) {
+  if (any( !is.character(self$StartFrom[1]) | length(self$StartFrom[1])==0 | ! nzchar(self$StartFrom[1]) )) {
```

Comment:
Let's check to see if the [1] indexing is needed here -- should not be needed for the length() statement. For is.character, and nzchar, could return multiple elements possibly.

Reply:
Conversation with Jeremy 8/2: Revert the || test here, but as above use [1] indexing to ensure just the first value used (except for `length()`).